tests/attr/test_lrp.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+
+import torch
+import torch.nn as nn
+
+from captum.attr import LRP, InputXGradient
+from captum.attr._utils.lrp_rules import (
+    Alpha1_Beta0_Rule,
+    EpsilonRule,
+    GammaRule,
+    IdentityRule,
+)
+
+from ..helpers.basic import BaseTest, assertTensorAlmostEqual
+from ..helpers.basic_models import (
+    BasicModel_ConvNet_One_Conv,
+    BasicModel_MultiLayer,
+    SimpleLRPModel,
+)
+
+
+def _get_basic_config():
+    input = torch.arange(16).view(1, 1, 4, 4).float()
+    return BasicModel_ConvNet_One_Conv(), input
+
+
+def _get_rule_config():
+    relevance = torch.tensor([[[-0.0, 3.0]]])
+    layer = nn.modules.Conv1d(1, 1, 2, bias=False)
+    nn.init.constant_(layer.weight.data, 2)
+    activations = torch.tensor([[[1.0, 5.0, 7.0]]])
+    input = torch.tensor([2, 0, -2])
+    return relevance, layer, activations, input
+
+
+def _get_simple_model(inplace=False):
+    model = SimpleLRPModel(inplace)
+    inputs = torch.tensor([[1.0, 2.0, 3.0]])
+
+    return model, inputs
+
+
+def _get_simple_model2(inplace=False):
+    class MyModel(nn.Module):
+        def __init__(self, inplace):
+            super().__init__()
+            self.lin = nn.Linear(2, 2)
+            self.lin.weight = nn.Parameter(torch.ones(2, 2))
+            self.relu = torch.nn.ReLU(inplace=inplace)
+
+        def forward(self, input):
+            return self.relu(self.lin(input))[0].unsqueeze(0)
+
+    input = torch.tensor([[1.0, 2.0], [1.0, 3.0]])
+    model = MyModel(inplace)
+
+    return model, input
+
+
+class Test(BaseTest):
+    def test_lrp_creator(self):
+        model, _ = _get_basic_config()
+        model.conv1.rule = 1
+        self.assertRaises(TypeError, LRP, model)
+
+    def test_lrp_creator_activation(self):
+        model, inputs = _get_basic_config()
+        model.add_module("sigmoid", nn.Sigmoid())
+        lrp = LRP(model)
+        self.assertRaises(TypeError, lrp.attribute, inputs)
+
+    def test_lrp_basic_attributions(self):
+        model, inputs = _get_basic_config()
+        logits = model(inputs)
+        _, classIndex = torch.max(logits, 1)
+        lrp = LRP(model)
+        relevance, delta = lrp.attribute(
+            inputs, classIndex.item(), return_convergence_delta=True
+        )
+        self.assertEqual(delta.item(), 0)
+        self.assertEqual(relevance.shape, inputs.shape)
```

Comment:
nit: Could we, please, also assert relevance values ?

Reply:
Added the value assertion